### PR TITLE
Create extension with placeholder module

### LINKED ISSUE #1: create a Slicer extension with a placeholder module to get started

### PR DESCRIPTION
**Pull request by ebrahimebrahim**
Closes #1 

* Use extension wizard to start the extension
* Create a scripted module "OpenLIFU Home" to serve as the starting point for the custom app
* Update README

Still need to 
- [x] Test that this can be installed
- [x] Test that this can be bundled with a custom application with its main module set as home module
- [x] Update README with instructions for building

### CODE REVIEW

File: OpenLIFUHome/OpenLIFUHome.py [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+
+class OpenLIFUHome(ScriptedLoadableModule):
+    """Uses ScriptedLoadableModule base class, available at:
+    https://github.com/Slicer/Slicer/blob/main/Base/Python/slicer/ScriptedLoadableModule.py
+    """
+
+    def __init__(self, parent):
+        ScriptedLoadableModule.__init__(self, parent)
+        self.parent.title = _("OpenLIFU Home")
+        self.parent.categories = [translate("qSlicerAbstractCoreModule", "OpenLIFU")]
```
Comment (ebrahimebrahim) [inline OpenLIFUHome/OpenLIFUHome.py:33]: Is the category OpenLIFU good?

This is what is meant by "category"
image

Comment (peterhollender) [reply OpenLIFUHome/OpenLIFUHome.py:33]: Yeah, that seems fine. I'd have to think about alternatives

Comment (sjh26) [reply OpenLIFUHome/OpenLIFUHome.py:33]: Yep, grouping them like this is fine

**Review by sjh26 — APPROVED** [review]
lgtm

**Pull request merged**